h5p.classes.php

```diff
@@ -1976,7 +1984,7 @@ public function filterParameters(&$content) {
         $content['slug'] = $this->generateContentSlug($content);
 
         // Remove old export file
-        $this->fs->deleteExport($content['id'] . '.h5p');
+        $this->fs->deleteExport(H5PExport::buildExportFileName($content));
```

Comment:
This doesn't seem right to me. The original code tried to delete the export file using the old naming (without slug)

Reply:
Hey Pal, thanks for your reply. I assumed this to be a bug, which is why I changed the logic here. Now that I know it's a backwards compat thing, this makes sense of course.
The question for me then is, why is the export file with the "new" name - in case it exists - not deleted here? Assuming we already have an H5P content element with an existing export file (according to the new naming logic) and are updating the content. Is the approach to simply overwrite it while generating an updated export file, and not explicitly delete it beforehand?